src/hooks/screen/useChatInfoHook.ts

```diff
@@ -78,7 +68,7 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   };
 
   const handleShowArrow = (item: Member) => {
-    if (params?.from === SIGNED) {
+    if (params?.from === SIGNED || params?.from === ANONYMOUS) {
       return item?.user?.userId !== signedProfileId;
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for `handleShowArrow` has been updated to check if `params?.from` is either `SIGNED` or `ANONYMOUS`. This seems to be a logical extension of the previous code, which only checked for `SIGNED`. However, please ensure that the rest of your codebase can handle this new condition properly, especially in places where `handleShowArrow` is used. If there are no issues with this, then the changes look good.


<!-- This is an auto-generated comment by OSS CodeRabbit -->